bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcConfiguration.java

```diff
@@ -99,11 +99,11 @@ public Builder setEnvironment(@NonNull Environment builderEnvironment) {
          * @return {@link BcmcConfiguration}
          */
         @NonNull
-        public BcmcConfiguration build() {
+        protected BcmcConfiguration buildInternal() {
             return new BcmcConfiguration(
-                    mBuilderShopperLocale,
-                    mBuilderEnvironment,
-                    mBuilderClientKey
+                    getMBuilderShopperLocale(),
```

Comment:
I see a lot of these `getM` fields, the reason is probably that the super classes are in java but the base class is in Kotlin. I think they look quite ugly 🙈  Maybe we should refactor the super classes before we do the base one, but that is probably better done in another PR. @caiofaustino what do you think?

Reply:
Hmmm on a second thought, the base class is now in kotlin and the `m` prefix is not a Kotlin syntax. Maybe if we refactor the field names in the `BaseConfigurationBuilder` and remove the `m`, the super java classes would become more readable:
`BaseConfigurationBuilder`: `mBuilderShopperLocale` -> `builderShopperLocale`
`BcmcConfiguration`: `getMBuilderShopperLocale()` -> `getBuilderShopperLocale()`

Reply:
I think my comment on `BaseConfigurationBuilder.kt` should do the trick, just having the base variables public.